### PR TITLE
Add llvm apache exception

### LINKED ISSUE #871: Add new Apache exception for LLVM
See https://lists.spdx.org/pipermail/spdx-legal/2017-December/002421.html

### PR DESCRIPTION
**Pull request by SaravananOffl**
fixes #871

### CODE REVIEW

File: src/licensedcode/data/licenses/apache-exception-llvm.LICENSE [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---- LLVM Exceptions to the Apache 2.0 License ----
+
+As an exception, if, as a result of your compiling your source code, portions
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/apache-exception-llvm.LICENSE]: Could you add also a RULE for the same text  minus the title line?

**Comment by SaravananOffl** [comment]
@pombredanne I've added the rules! :+1:

File: src/licensedcode/data/licenses/apache-exception-llvm.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+ +key: apache-exception-llvm
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/apache-exception-llvm.yml]: IMHO we may need a better name.
@DennisClark what would this be? `llvm-apache-exception`

**Comment by pombredanne** [comment]
```
#############################
INVALID LICENSE YAML FILE: c:\projects\scancode-toolkit\src\licensedcode\data\licenses\apache-exception-llvm.yml
#############################
```
@SaravananOffl your yaml file is not valid... see the CI build logs.

File: src/licensedcode/data/licenses/apache-exception-llvm.yml (hunk 2 above)
Comment (pombredanne) [reply src/licensedcode/data/licenses/apache-exception-llvm.yml]: @SaravananOffl All your keys are prefixed with a `+` .... invalid YAML!

Comment (SaravananOffl) [reply src/licensedcode/data/licenses/apache-exception-llvm.yml]: @pombredanne should i change the name also ?

Comment (pombredanne) [reply src/licensedcode/data/licenses/apache-exception-llvm.yml]: @SaravananOffl you can fix the file content, but keep the name for now until we hear back from @DennisClark

**Comment by SaravananOffl** [comment]
@pombredanne Is this fine or anything more should be done ?.

**Comment by pombredanne** [comment]
@SaravananOffl We are all good here. Thank ++. There is an heisenbug on the CI but that not an issue for you https://ci.appveyor.com/project/nexB/scancode-toolkit/build/1486/job/1a6quok45k9dsnvp#L306

I am just waiting for @DennisClark feedback on the actual name to pick for this. You are done on your side. And this is great!

**Comment by pombredanne** [comment]
Looking good! thanks!

**Comment by pombredanne** [comment]
Actually I did not review your commit messages which could have been made better.

And I absolutely need a signoff from you at least in this PR.
Can you add a message with it to this PR?

**Pull request merged**